docs/content/en/docs/implementing/evaluatemetrics.md

```diff
@@ -103,6 +103,28 @@ $ kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta2/namespaces/podtato-kube
 }
 ```
 
+## Querying Metrics over a Timerange
```

Comment:
Can we please describe the default behavior when the `.spec.range.interval` is not set ? Otherwise looks good :)